cirq-web/cirq_ts/dist/index.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
```

Comment:
I think this should be a front page for all the development files, i.e. a "table of contents" or "index" for "bloch sphere", "3d circuit type 1", "3d circuit type 2" , etc.  - but we can do that in the next PR as we add more widgets.